WP_Requirements_Check.php

```diff
@@ -140,9 +140,10 @@ protected static function _php_at_least( $min_version ) {
 	 * @access public
 	 */
 	public function php_version_notice() {
+		$message = apply_filters( 'wp_requirements_check_php_notice', 'The &#8220;%s&#8221; plugin cannot run on PHP versions older than %s. Please contact your host and ask them to upgrade.' );
```

Comment:
Ideally, filters would be documented similarly to how WordPress core does it: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

I'd probably pass the already sprintf'd message to the filter, and add `esc_html( $this->title )` and `$this->php` as additional parameters. This way, the message can be changed depending on the actual version.

Also, I'd use `%1$s` and `%2$s` instead of just `%s` so that you can re-arrange the values when changing the string.

Reply:
Example:

Reply:
Good points. I'll fix it soon